graph/pull_v2.go

```diff
@@ -368,6 +368,26 @@ func (p *v2Puller) verifyTrustedKeys(namespace string, keys []libtrust.PublicKey
 }
 
 func (p *v2Puller) validateManifest(m *manifest.SignedManifest, tag string) (verified bool, err error) {
+	// If pull by digest, then verify the manifest digest.
```

Comment:
Seems like in the original version this was indeed supposed to be in the beginning, not sure if that matters though.